src/codegen/build_module.cc

```diff
@@ -882,6 +887,16 @@ TVM_REGISTER_GLOBAL("_GetCurrentTarget")
   *ret = Target::Current(allow_not_defined);
   });
 
+TVM_REGISTER_GLOBAL("_GetLLVMTargetHost")
+.set_body([](TVMArgs args, TVMRetValue* ret) {
+  *ret = Target::GetLLVMTargetHost();
+});
+
+TVM_REGISTER_GLOBAL("_SetLLVMTargetHost")
```

Comment:
It is a bit bad to use global state to pass this argument around. Would be great to pass this argument as an argument of PackToLLVM instead.


I wonder if it is necessary to set the target to be the specific target, or can we directly use a somewhat generic target (this means LLVM won't perform any target specific optimization which is fine here)